nvbench/detail/state_exec.cuh

```diff
@@ -23,10 +23,12 @@
        "Do not include it directly."
 #endif // NVBENCH_STATE_EXEC_GUARD
 
+#include <nvbench/config.cuh>
 #include <nvbench/exec_tag.cuh>
 #include <nvbench/state.cuh>
 
 #include <nvbench/detail/kernel_launcher_timer_wrapper.cuh>
+#include <nvbench/detail/measure_cupti.cuh>
```

Comment:
When I run `nvbench.example.auto_throughput` with CUPTI disabled, I get

```
bin/nvbench.example.auto_throughput: symbol lookup error: bin/nvbench.example.auto_throughput: undefined symbol: nvbench::detail::measure_cupti_base::generate_summaries()
```

Wrapping this header in `#ifdef NVBENCH_HAS_CUPTI / #endif` fixes it.